easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.8-foss-2018b-R-3.5.1.eb

```diff
@@ -0,0 +1,987 @@
+easyblock = 'Bundle'
+
+name = 'R-bundle-Bioconductor'
+version = '3.8'
+biocver = version
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'http://www.r-project.org/'
+description = """R is a free software environment for statistical computing and graphics."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+dependencies = [
+    ('R', '3.5.1'),
+    ('HDF5', '1.10.2'),  # for rhdf5
+    ('MariaDB-connector-c', '2.3.7'),  # for RMySQL
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/3.8/bioc/src/contrib/',
+        'https://bioconductor.org/packages/3.8/bioc/src/contrib/Archive/%(name)s',
+        'https://bioconductor.org/packages/3.8/data/annotation/src/contrib/',
+        'https://bioconductor.org/packages/3.8/data/experiment/src/contrib/',
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+ext_options = exts_default_options
+bioconductor_options = exts_default_options
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+# CRAN packages on which these Bioconductor packages depend are available in R module on which this depends
+# !! order of packages is important !!
+# packages updated on Sept 25th 2018
+exts_list = [
+    ('BiocGenerics', '0.28.0', {
+        'checksums': ['68e5a2737f847dd49c6cbd60a77291fd48b39fd219effef0a5c7eaf360d67733'],
+    }),
+    ('Biobase', '2.42.0', {
+        'checksums': ['36729cf3fa2cf42cc64c7f8f4afe5d0dfc64cc860baea721c2741739b335d982'],
+    }),
+    ('S4Vectors', '0.20.1', {
+        'checksums': ['ef379329429f638f954d76c023d8509a8476bcbc824e067517c8767b0ccf90a3'],
+    }),
+    ('IRanges', '2.16.0', {
+        'checksums': ['81d9a0687665f6dacb970190266358c3ac1046e86d55c49e38b18760a6be5752'],
+    }),
+    ('GenomeInfoDbData', '1.2.0', {
+        'checksums': ['ac8c370b85021786ef67d9fdd1872930368abd2a46593147320e7b7d31b52636'],
+    }),
+    ('bitops', '1.0-6', {
```

Comment:
@smoors Why are you including R packages here that are already included in the `R` easyconfig that is used as a dependency?

Reply:
Good question.. I think this is what the easy_update tool spitted out. Should have checked this more carefully.

Reply:
Yes, that's almost certainly an `easy_update` artifact. The upstream uses combined `easyconfigs` that include both in one, I believel.